### DISCUSSION TITLE
is it possible to create a ISO for regolith3 based on ubuntu 22.04 ?

### DISCUSSION THREAD

**Discussion by Mokkujin** [regolith-linux/regolith-desktop#970] [Ideas]
you offer an image for Regolith 2.2 based on ubuntu 22.10. Would it be possible to offer an ISO for Regolith 3 based on Ubuntu 22.04 LTS, include your tweaks? GDM3 snapd and so on ... Ubuntu 22.04 LTS is a long term support ( i know u know ;-) ) so i think you got more dl's ... and make my life a little bit easier ;)

**Comment by strazto** [comment]
https://github.com/orgs/regolith-linux/discussions/898#discussioncomment-6920442 

> Due to time required to develop and test, we only ship ISO distribution of Regolith (AKA "Regolith Linux") on LTS releases

They mention that they only ship the distro on LTS releases, but I'm pretty sure they're referring to LTS releases of the underlying distro, rather than regolith, so there wouldn't be a problem with shipping regolith for Ubuntu 22.10
